tests/specs/main/logs.spec.js

```diff
@@ -66,3 +68,33 @@ test.describe('Logs actions', () => {
         }
     });
 });
+
+test.describe('file import/export', () => {
+    test('import logs from file', async ({page, baseURL}) => {
+        await page.goto(baseURL);
+        const directory = 'tests/data/main';
+        const fileName = 'import.json';
+        const fileWithPath = path.join(directory, fileName);
+        const [fileChooser] = await Promise.all([
+            page.waitForEvent('filechooser'),
+            page.getByLabel('Import logs').click()
+        ]);
+
+        await fileChooser.setFiles([fileWithPath]);
+        await expect(page.getByRole('heading', {name: 'Logs:'})).toHaveText('Logs: 2');
```

Comment:
is this check enough? it says only that "something" has loaded

the same with export

at least it's necessary to make sure exported data is identical to the original one before import

Reply:
ok, added more asserts